lib/main_common.pm

```diff
@@ -1474,6 +1474,7 @@ sub load_extra_tests_desktop {
     }
     if (gnomestep_is_applicable()) {
         loadtest "x11/remote_desktop/vino_screensharing_available";
+	loadtest "x11/rrdtool_x11" ;
```

Comment:
I think you need to move this out of the if :)

Reply:
ok, trying now.